src/subscription/checkout/monthly-billing-notification/MonthlyBillingNotification.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useIntl, defineMessages } from '@edx/frontend-platform/i18n';
+import { detailsSelector } from '../../data/details/selectors';
+
+const messages = defineMessages({
+  'subscription.checkout.billing.notification': {
+    id: 'subscription.checkout.billing.notification',
+    defaultMessage: 'You’ll be charged {price} USD on MM/DD then every 31 days until you cancel your subscription.',
```

Comment:
should we calculate `MM/DD` by adding 7 days to current date?
what will be the message in case of resubscribe? 

or we can just change the notification to be more generic not mentioning dates. 

e.g. "You be charged $111.0 USD after 7 days and then every 31 days until you cancel your subscription."

Reply:
This is actually a good option, I will add this in the discussion in Slack